prov/efa/src/rxr/rxr_rma.h

```diff
@@ -47,7 +47,7 @@ struct rxr_tx_entry *
 rxr_rma_alloc_readrsp_tx_entry(struct rxr_ep *rxr_ep,
 			       struct rxr_rx_entry *rx_entry);
 
-size_t rxr_rma_post_shm_rma(struct rxr_ep *rxr_ep,
+size_t rxr_rma_post_shm_write(struct rxr_ep *rxr_ep,
```

Comment:
I think this declaration can be deleted because the function ```rxr_rma_post_shm_write``` was only used inside ```rxr_rma.c```

Reply:
Yes, thanks.